src/js/components/aboutTheDataSidebar/AboutTheDataListView.jsx

```diff
@@ -16,6 +17,11 @@ const AboutTheDataListView = ({ section, selectItem }) => {
     const clickHandler = (e, index, section) => {
         e.preventDefault();
         selectItem(index, section);
+        Analytics.event({
+            category: 'About the Data',
+            action: `Clicked ${section}`,
```

Comment:
you could try doing a console log of section.  I think it's an object not a string.  And that might not have the info you want,  We want the action to include the term name not the section.